package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ot-telemetry-collector",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Let's push it to the npm repository after merge

Reply:
yes, i planned to do that